pbo/src/discrete.py

```diff
@@ -0,0 +1,154 @@
+# Custom imports
+from pbo.src.base import *
+
+###############################################
+### Class discrete
+### A PBO agent for discrete optimization
+class discrete(base_pbo):
+    def __init__(self, params, act_dim, obs_dim):
+
+        # Initialize from arguments
+        self.pdf        = params.pdf
+        self.act_dim    = len(act_dim) # number of parameters
+        self.obs_dim    = obs_dim
+        self.param_dim  = act_dim # list containing the dimension for each param
+        self.n_gen      = params.n_gen
+        self.n_ind      = params.n_ind
+        self.size       = self.n_gen*self.n_ind
+        self.n_cpu      = params.n_cpu
+
+        self.param_gen        = params.param_gen
+        self.batch            = params.batch
+        self.lr               = params.lr
+        self.epochs           = params.epochs
+        self.adv_clip         = params.adv_clip
+        self.adv_decay        = params.adv_decay
+        self.continuity_regul = params.continuity_regul
+
```

Comment:
@aurelegoetz I could not find a provided value of continuity_regul in the discrete json example file, could you provide it ?

Reply:
I'll provide it in the next commit.